scripts/0_wipe.sql

```diff
@@ -0,0 +1 @@
+DROP DATABASE %s;
```

Comment:
Question: Why make this a Go `fmt` string rather than valid SQL? Also, why does this logic not fit in the `1_clean.sql` script?

Reply:
> Why make this a Go fmt string rather than valid SQL?
I did not want to hard-code the database name here; the db name is from env var.  Also, the sql lib won't take `$1` in that spot, it only uses `$#` in SELECT (or equivalent) statements.

> why does this logic not fit in the 1_clean.sql script
By splitting up the `DROP DATABASE` from the `1_clean.sql`, the _first_ run of `DB.Init` follows a different code path than each subsequent invocation, achieving a wee bit of better coverage.  It increased my confidence, at least.

Reply:
I don't mind having it one place or the other. It's just different than the other scripts by effectively requiring a parameter to be given. This wouldn't work with the `RunScript()` method of the DB.